ROOTFS/etc/waggle/sanity/warning/k3s_docker_overlay.test

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+echo "Docker-K3S Overlay Test: Begin"
+
+dockerSym=$(find /var/lib/ -type l -ls | grep docker)
+racherFSym=$(find /etc/ -type l -ls | grep rancher)
+kubeletSym=$(find /var/lib/ -type l -ls | grep kubelet)
+rancherSSym=$(find /var/lib/ -type l -ls | grep rancher)
+
+if [ -z "$dockerSym" ]; then
```

Comment:
You can just do 
`if [ -h /var/lib/docker ]`

`-h` checks if the file is a symlink